examples/rpc/README.md

```diff
@@ -0,0 +1,33 @@
+# RPC Demo
+
+A very simple example to show peer to peer communication with Godot's high level 
+
+multiplayer API calls.

```

Comment:
Extra newlines should be removed so the lines aren't interpreted as separate paragraphs.